src/test/java/it/pagopa/pn/deliverypush/externalclient/pnclient/safestorage/PnSafeStorageClientImplTestIT.java

```diff
@@ -28,12 +26,10 @@
 import static org.mockserver.model.HttpResponse.response;
 
 
-@ExtendWith(SpringExtension.class)
```

Comment:
Inutile aggiunegre il contesto di Spring in quanto i metodi di questa classe di test eseguono solo test unitari.